src/components/Randombeer.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useState, useEffect } from 'react';
+import Header from './Header';
+import axios from 'axios';
+
+function Randombeer() {
+  const [random, setRandom] = useState({});
+  const [randomBeerDetail, setRandomBeerDetail] = useState({});
+
+  const getFromApi = async () => {
+    try {
+      const apiCall = await axios.get(`https://ih-beers-api2.herokuapp.com/beers/${random}`);
```

Comment:
Here you should've used the link to random, like:

``const apiCall = await axios.get(`https://ih-beers-api2.herokuapp.com/beers/random`);``